forge-gui/res/cardsfolder/upcoming/jade_orb_of_dragonkind.txt

```diff
@@ -0,0 +1,10 @@
+Name:Jade Orb of Dragonkind
+ManaCost:2 G
+Types:Artifact
+A:AB$ Mana | Cost$ T | Produced$ G | TriggersWhenSpent$ TrigSpent | SpellDescription$ Add {G}. When you spend this mana to cast a Dragon creature spell, it enters the battlefield with an additional +1/+1 counter on it and gains hexproof until your next turn. (It can't be the target of spells or abilities your opponents control.)
+SVar:TrigSpent:Mode$ SpellCast | ValidCard$ Creature.Dragon | ValidActivatingPlayer$ You | Execute$ TrigETBReplacement | TriggerDescription$ When you spend this mana to cast a Dragon creature spell, it enters the battlefield with an additional +1/+1 counter on it and gains hexproof until your next turn. (It can't be the target of spells or abilities your opponents control.)
+SVar:TrigETBReplacement:DB$ Pump | Defined$ TriggeredCard | KW$ etbCounter:P1P1:1:no Condition:no desc | PumpZone$ Stack | SubAbility$ DBPump
+SVar:DBPump:DB$ Pump | Defined$ TriggeredCard | Duration$ UntilYourNextTurn | KW$ Hexproof | PumpZone$ Stack
```

Comment:
Not sure I love the Pump -> Pump but the durations seem like they ought to be different

Reply:
Maybe Effect + MovedReplacement + PutCounter would be better

Reply:
Works for me – I'll wire it up

Reply:
yeah it needs to be a Move Replacement

the keyword etCounter will be removed someday